keyboards/maartenwut/eon95/info.json

```diff
@@ -0,0 +1,338 @@
+                {"x":14, "y":1.25},
+                {"x":15.5, "y":1.25},
+                {"x":16.5, "y":1.25},
+                {"x":17.5, "y":1.25},
+                {"x":18.5, "y":1.25},
+                
+                {"x":0, "y":2.25, "w":1.5},
+                {"x":1.5, "y":2.25},
+                {"x":2.5, "y":2.25},
+                {"x":3.5, "y":2.25},
+                {"x":4.5, "y":2.25},
+                {"x":5.5, "y":2.25},
+                {"x":6.5, "y":2.25},
+                {"x":7.5, "y":2.25},
+                {"x":8.5, "y":2.25},
+                {"x":9.5, "y":2.25},
+                {"x":10.5, "y":2.25},
+                {"x":11.5, "y":2.25},
+                {"x":12.5, "y":2.25},
+                {"x":13.5, "y":2.25, "w":1.5},
+                {"x":15.5, "y":2.25},
+                {"x":16.5, "y":2.25},
+                {"x":17.5, "y":2.25},
+                {"x":18.5, "y":2.25},
+                
+                {"x":0, "y":3.25, "w":1.75},
+                {"x":1.75, "y":3.25},
+                {"x":2.75, "y":3.25},
+                {"x":3.75, "y":3.25},
+                {"x":4.75, "y":3.25},
+                {"x":5.75, "y":3.25},
+                {"x":6.75, "y":3.25},
+                {"x":7.75, "y":3.25},
+                {"x":8.75, "y":3.25},
+                {"x":9.75, "y":3.25},
+                {"x":10.75, "y":3.25},
+                {"x":11.75, "y":3.25},
+                {"x":12.75, "y":3.25},
+                {"x":13.75, "y":3.25, "w":1.25},
+                {"x":15.5, "y":3.25},
+                {"x":16.5, "y":3.25},
+                {"x":17.5, "y":3.25},
+                {"x":18.5, "y":3.25},
+                
+                {"x":0, "y":4.25, "w":1.25},
+                {"x":1.25, "y":4.25},
+                {"x":2.25, "y":4.25},
+                {"x":3.25, "y":4.25},
+                {"x":4.25, "y":4.25},
+                {"x":5.25, "y":4.25},
+                {"x":6.25, "y":4.25},
+                {"x":7.25, "y":4.25},
+                {"x":8.25, "y":4.25},
+                {"x":9.25, "y":4.25},
+                {"x":10.25, "y":4.25},
+                {"x":11.25, "y":4.25},
+                {"x":12.25, "y":4.25, "w":1.75},
+                {"x":14.25, "y":4.5},
+                {"x":15.5, "y":4.25},
+                {"x":16.5, "y":4.25},
+                {"x":17.5, "y":4.25},
+                {"x":18.5, "y":4.25, "h":2},
+                
+                {"x":0, "y":5.25, "w":1.25},
+                {"x":1.25, "y":5.25, "w":1.25},
+                {"x":2.5, "y":5.25, "w":1.25},
+                {"x":3.75, "y":5.25, "w":6.25},
+                {"x":10, "y":5.25},
+                {"x":11, "y":5.25},
+                {"x":12, "y":5.25},
+                {"x":13.25, "y":5.5},
+                {"x":14.25, "y":5.5},
+                {"x":15.25, "y":5.5}
+                {"x":16.5, "y":5.25},
+                {"x":17.5, "y":5.25},
+            ]
+        },
+        "LAYOUT_ansi": {
```

Comment:
This and the ISO layout are missing a key vs the LAYOUT macros. Split backspace?